README.md

```diff
@@ -78,6 +78,7 @@ Initialize sdk in method `onCreate()`.
 protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     FacebookSdk.sdkInitialize(getApplicationContext());
```

Comment:
Yeah, I know, that's why I put a comment on line 100 saying // <--- add  

Just like the one you have [here](https://github.com/facebook/react-native-fbsdk/blame/master/README.md#L93) for the import.

Anyway, just to be clear, you want me to delete line 99 of this PR, right? 

Another solution would be to switch the syntax of the snippet to a diff. which makes it way more clear ( but you lose the Java syntax highlight )

That would look like this:  

![screen shot 2016-05-17 at 6 05 21 pm](https://cloud.githubusercontent.com/assets/1247834/15340932/0aeb748c-1c5a-11e6-8773-d46a9cd469d8.png)

Whatever you prefer is ok to me.